qiita_db/metadata_template.py

```diff
@@ -1051,6 +1085,27 @@ def get_filepaths(self, conn_handler=None):
 
         return [(fpid, base_fp(fp)) for fpid, fp in filepath_ids]
 
+    def categories(self):
+        """Get the categories associated with self
+
+        Returns
+        -------
+        set
```

Comment:
Is it conceivable that column order might matter in some contexts? `information_schema.columns` incldues an `ordinal_position` column that could potentially be used to provide the order in an output `list` instead of an unordered `set`